drivers/srmetadata.py

```diff
@@ -93,35 +89,65 @@ def open_file(path, write=False):
     return file_p
 
 
-def file_write_wrapper(fd, offset, blocksize, data, length):
+# Writes data to a file at a given offset. Padding (consisting of spaces) may
```

Comment:
Should this not be in a method comment, i.e """ XXXX """ under the method signature?

Reply:
Yeah - I wasn't sure what the convention was.